keyboards/gboards/gergoplex/gergoplex.c

```diff
@@ -0,0 +1,80 @@
+#include "gergoplex.h"
+
+bool         i2c_initialized = 0;
+i2c_status_t mcp23018_status = 0x20;
+
+void matrix_init_kb(void) {
+    // (tied to Vcc for hardware convenience)
+    // DDRB  &= ~(1<<4);  // set B(4) as input
+    // PORTB &= ~(1<<4);  // set B(4) internal pull-up disabled
+
+    // unused pins - C7, D4, D5, D7, E6
+    // set as input with internal pull-up enabled
+    DDRB &= ~(1 << 0 | 1 << 4 | 1 << 5 | 1 << 6 | 1 << 7);
```

Comment:
I intend to replace this according to [this comment](https://github.com/qmk/qmk_firmware/pull/8928/files#r449149710):

> We rather use the GPIO control functions: https://docs.qmk.fm/#/internals_gpio_control?id=functions